docs/ReferenceArrayInput.md

```diff
@@ -46,13 +46,13 @@ import { ReferenceArrayInput, SelectArrayInput } from 'react-admin';
 | Prop               | Required | Type                                        | Default                            | Description                                                                                                         |
 |--------------------|----------|---------------------------------------------|------------------------------------|---------------------------------------------------------------------------------------------------------------------|
 | `filter`           | Optional | `Object`                                    | `{}`                               | Permanent filters to use for getting the suggestion list                                                            |
+| `label`            | Optional | `string`                                    | -                                  | A custom label can be used when `ReferenceArrayInput` is in a Filter array.                         |
```

Comment:
I would rather formulate this by saying this is useful **only** when `ReferenceArrayInput` is in a Filter array